### ISSUE TITLE
How to check the gradients of lora layers when training a peft model

### ISSUE THREAD

**Issue by stardusts-hj** [huggingface/peft#1308]
### Feature request

when  I trained a lora model like this
```python
model = get_peft_model(model, lora_config)
training(model,data)
```
How can I check the gradients of lora layers from a `peft` model ?

### Motivation

check gradients of lora layers from peft model during training

### Your contribution

ni

**Comment by BenjaminBossan** [comment]
What exactly do you want to check? In general, this is just a PyTorch model and you can go through the parameters as usual. Note that LoRA layers have names that look something like:

`model.transformer.h[0].self_attention.query_key_value.lora_A["default"].weight`

Note the **lora_A["default"].weight** part of the name, which indicates this is the default adapter for the LoRA A-weight. The exact name of the parameter is depends on the model architecture you're using but to get a quick overview, you can run something like:

```python
lora_params = {n: p for n: p in model.named_parameters() if "lora_" in n}
```

**Comment by stardusts-hj** [comment]
> What exactly do you want to check? In general, this is just a PyTorch model and you can go through the parameters as usual. Note that LoRA layers have names that look something like:
> 
> `model.transformer.h[0].self_attention.query_key_value.lora_A["default"].weight`
> 
> Note the **lora_A["default"].weight** part of the name, which indicates this is the default adapter for the LoRA A-weight. The exact name of the parameter is depends on the model architecture you're using but to get a quick overview, you can run something like:
> 
> ```python
> lora_params = {n: p for n: p in model.named_parameters() if "lora_" in n}
> ```

Thank you!
